demisto_sdk/__main__.py

```diff
@@ -912,11 +912,9 @@ def id_set_command(**kwargs):
 
 
 # ====================== merge-id-sets ====================== #
-@main.command(name='merge-id-sets',
-              hidden=True,
-              short_help='Merge two id_sets')
-@click.help_option(
-    '-h', '--help'
```

Comment:
why remove the help options?

Reply:
We don't need to specify it. `--help` will always be there, even without it.
The help option will only enable `-h`, an option that is not needed.